docs-new/.github/bin/deploy_docs.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+while [[ "$#" -gt 0 ]]; do
+    case $1 in
+        -v|--version) ICEBERG_VERSION="$2"; shift ;;
+        *) echo "Unknown parameter passed: $1"; exit 1 ;;
+    esac
+    shift
+done
+
+GIT_BRANCH="docs-${ICEBERG_VERSION}"
+
+# change to branch
+git checkout -b $GIT_BRANCH
+
+#remove all local files and leave site/ dir
+rm ./*
```

Comment:
How does this leave the `site` dir?

Reply:
Won't remove directories without -R

Reply:
Is it leaving all the directories then?

Reply:
Yes, and I delete those a few lines below.